testcases/MachineConfig.py

```diff
@@ -54,6 +55,9 @@ def setUp(self):
             self.system_name = conf.args.system_name
             self.cv_HMC = self.cv_SYSTEM.hmc
             self.lpar_prof = conf.args.lpar_prof
+            self.c = self.cv_HMC.get_host_console()
+            self.mmulist = self.c.run_command("tail /proc/cpuinfo | grep MMU")
+            self.mmu = str(self.mmulist[0]).split(':')[1].strip()
```

Comment:
please move all these sections to OsClass, as this is not needed to put in setup